react-api-client/src/camera/useCameraSettings.ts

```diff
@@ -0,0 +1,63 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { createCameraSettings } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type {
+  UseMutateFunction,
+  UseMutationOptions,
+  UseMutationResult,
+} from 'react-query'
+import type {
+  CameraResponse,
+  CameraSettings,
+  ErrorResponse,
+} from '@opentrons/api-client'
+import type { CameraId } from '@opentrons/shared-data'
+
+export type UseUpdateCameraMutationResult = UseMutationResult<
+  CameraResponse,
+  AxiosError<ErrorResponse>,
+  CameraSettings
+> & {
+  updateCameraSettings: UseMutateFunction<
+    CameraResponse,
+    AxiosError<ErrorResponse>,
+    CameraSettings
+  >
+}
+
+export function useCameraSettings(
```

Comment:
```suggestion
export function useCreateCameraSettings(
```
We generally use some sort of "create" verbiage in front of hooks that denote an underlying `POST` request.